app/src/main/java/com/ddoczi/tasky/agenda/presentation/home/HomeScreen.kt

```diff
@@ -108,21 +114,32 @@ fun HomeScreen(
             )
             Spacer(modifier = Modifier.height(24.dp))
             LazyColumn(modifier = Modifier.fillMaxWidth()){
-                items(state.agendaItems) {
+                items(state.agendaItems) { agendaItem ->
                     HomeAgendaItem(
-                        item = it,
-                        color = when(it) {
+                        item = agendaItem,
+                        color = when(agendaItem) {
                                  is AgendaItem.Event -> LightGreen
                                  is AgendaItem.Task -> Green
                                  is AgendaItem.Reminder -> Light
                                },
-                        onItemOptionsClick = {onEvent(HomeEvent.OnItemOptionsClick(it))},
-                        onItemClick = { onEvent(HomeEvent.OnItemClick(it)) }
+                        onItemOptionsClick = {onEvent(HomeEvent.OnItemOptionsClick(agendaItem))},
+                        onItemClick = { onEvent(HomeEvent.OnItemClick(agendaItem)) }
                     )
                     Spacer(modifier = Modifier.height(16.dp))
                 }
             }
-
+            Box(
+                modifier = Modifier.fillMaxWidth(),
+                contentAlignment = Alignment.CenterEnd) {
+                TaskyDropdown(
+                    items = AgendaOption.entries.map { it.name.lowercase().replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() } },
```

Comment:
Isn't the if/else unnecessary? If it's not a lowercase, it must be an uppercase, so nothing would happen in that case. So the code in the `replaceFirstChar` could be shortened to `it.titlecase()`

I don't think you can create an extension for the enum class itself independent of its values, but you could of course define one for the list of entries:
```
val <T : Enum<T>> List<Enum<T>>.names: List<String> 
    get() = map { it.name.replaceFirstChar { it.titlecase() } }
```
Then you can do
```
println(AgendaOption.entries.names)
```